### ISSUE TITLE
Android resizing documentation change

### ISSUE THREAD

**Issue by ljukas** [lodev09/react-native-true-sheet#89]
Android implementation using just `['auto']` will not resize the sheet when the content changes, you need to use the following sizes `['auto', 'large']` for it to resize when content changes.

This is not in the documentation that I can find, I instead found it in an old issue. Would be nice if this was added to troubleshooting section of the documentation

**Comment by lodev09** [comment]
Yeah, I was dealing with this issue on android but got distracted -- so this qualifies as a bug so it's no in the documentation. I'll revisit this when I get a chance.

**Comment by LahmerIlyas** [comment]
@lodev09 any plans to fix this issue?

**Comment by deepakkumardk** [comment]
it's a weird bug, totally contradicts the documentation, and it's been around a year and no update on the issue!!
@lodev09

**Comment by lodev09** [comment]
Here's the fix for this #193
